modules/test/playwright/tests/object-web/main/objectDefinitions.spec.ts

```diff
@@ -29,6 +30,7 @@ export const test = mergeTests(
 	collectionsPagesTest,
 	dataApiHelpersTest,
 	featureFlagsTest({
+		'LPD-17564': {enabled: true},
```

Comment:
We can isolate the use of this FF for testing this story, I believe this should be the standard we should follow from now on to ensure that current changes are not breaking existing tests without the use of FF.

Reply:
sure!